Projects/Marryting/Sources/AppDelegate.swift

```diff
@@ -1,4 +1,5 @@
 import UIKit
+import GuestList
 import DesignSystem
```

Comment:
P3
DesignSystem 임포트도 잘못했었네요. 아이쿠.ㅜ 이 부분은 제가 반영하겠습니다.
```suggestion
import DesignSystem
import GuestList
import UIKit
```